sections/Chapter 3 Sprint 1/postgreSQL.tex

```diff
@@ -0,0 +1,41 @@
+\subsection{PostgreSQL}
+The group last year chose to implement the database using PostgreSQL which is an relational database system \cite{knox2020}.
+
+A relational database is a collection of tables, known as relations. Relational databases are used to provide access to related data points.
```

Comment:
I miss a mathematical describtion of relations. Something ala "A relational database can be described mathematically as a collection of relations containing tuples. Each relation contains attributes describing the domain of the values of the tuples. A relational database describes these relations in the form of a table with columns corresponding to the name and domain of the attributes of the represented relation attributes. The rows of the table correspond to tuple values of the relation.
In the context of an enterprise......"